go-apps/meepctl/utils/kibana.go

```diff
@@ -112,9 +117,9 @@ func isKibanaReady(cobraCmd *cobra.Command) bool {
 }
```

Comment:
This error is never reported.
Try to adopt the 
if err != nil {
do something
}
else do something else 

structure from now on